.github/workflows/ci.yml

```diff
@@ -52,38 +57,87 @@ jobs:
           # Make sure the database is actually working
```

Comment:
Given the health-check, can we replace the `sleep 5` above here with something that actually waits for the containers to be ready?

Reply:
Apparently docker-compose only respects/runs health checks for a service's dependencies, I tried making a dummy service that just depended on the other two, but it didn't run. Any ideas?

Reply:
Ah, reading the docs sounds like the health-check is purely a debug status thing, it's not used by docker-compose in any way.

Might be good to use something like `wait-for`, but not necessary right now.